src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+import java.util.ArrayList;
+
+//an association class related Tag,Person
+public class Tagging {
+	private Person person;
+	private Tag tag;
+	private TagOperationFlag operation;
+
+	/**
+	 * Common container for all tagging association
+	 */
+	private static ArrayList<Tagging> TaggingList;
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class.